sys/net/gnrc/network_layer/ipv6/nib/Kconfig

```diff
@@ -57,8 +57,23 @@ config GNRC_IPV6_NIB_DNS
 
 config GNRC_IPV6_NIB_ADV_ROUTER
     bool "Activate router advertising at interface start-up"
+    default n if USEMODULE_GNRC_DHCPV6_CLIENT_IA_PD || USEMODULE_GNRC_UHCPC
+    default n if USEMODULE_GNRC_IPV6_AUTO_SUBNETS
     default y if GNRC_IPV6_NIB_ROUTER && (!GNRC_IPV6_NIB_6LR || GNRC_IPV6_NIB_6LBR)
 
+config GNRC_IPV6_NIB_ADD_RIO_IN_LAST_RA
+    bool "Include a Route Information Option for subnets"
+    default y if USEMODULE_GNRC_IPV6_AUTO_SUBNETS
```

Comment:
Mhhh for that we would need dependency handling in Kconfig for GNRC ;-). We don't have that yet. Setting MODULE_GNRC_IPV6_NIB_RIO only would set the symbol MODULE_GNRC_IPV6_NIB_RIO to true AFAIK. Also, the right way to do it with Kconfig would be `depends`, otherwise you get cascading effects ;-).

Reply:
But maybe @leandrolanzieri can clear this up.